### PR TITLE
chore: ci scan password pattern

### PR DESCRIPTION
**Pull request by wenytang-ms**

### CODE REVIEW

File: .github/detect/regexes.json [hunk 1]
```diff
@@ -0,0 +1,48 @@
+{
+    "Slack Token": "(xox[p|b|o|a]-[0-9]{12}-[0-9]{12}-[0-9]{12}-[a-z0-9]{32})",
+    "Generic Private Key POST Encapsulation Boundary": " PRIVATE KEY-----",
+    "Generic Private Key Block POST Encapsulation Boundary": " PRIVATE KEY BLOCK-----",
+    "Private key": "-----BEGIN PRIVATE KEY-----",
+    "RSA private key": "-----BEGIN RSA PRIVATE KEY-----",
```
Comment (swatDong) [inline .github/detect/regexes.json:6]: Just noticed this PR and would like to callout - we do have private key in localhost certificate unit test. Any action to do to exclude such unit test?

Comment (wenytang-ms) [reply .github/detect/regexes.json:6]: You can put these exception files in excludes.txt

Comment (LongOddCode) [reply .github/detect/regexes.json:6]: How about use annotation or comment around target code to evade inspection

Comment (wenytang-ms) [reply .github/detect/regexes.json:6]: It’s better to use documents for explanation than annotation/comments, especially when introducing anything about passwords.

**Review by LongOddCode — APPROVED** [review]

**Pull request merged**